static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -78,6 +79,16 @@ const onCloseCmp = (): Promise<void> =>
         })
         .then(removeCmp);
 
+const onErrorCmp = (error: Error): void => {
+    reportError(
```

Comment:
I guess this is the integration with sentry?

Reply:
Yes that's right